portfolio/src/main/webapp/script.js

```diff
@@ -87,6 +87,7 @@ function createListElement(text) {
     return liElement;
 }
 
+/** Refetch comments after all comments are deleted */
```

Comment:
Oh sorry to clarify I meant that as an add on to the main functionality. Example:
/** Delete all comments then refresh comments list in UI. */